### ISSUE TITLE
Test AppVeyor Build

### ISSUE THREAD

**Issue by dantarion** [dantarion/SF5DInput#12]
Please test this automated AppVeyor Build...
https://github.com/dantarion/SF5DInput/releases/tag/v0.0.7

**Comment by michaelmarcuccio** [comment]
This build still has the memory error reference bug that was fixed by Wydd. Don't know if that was the question here or not.

**Comment by WydD** [comment]
Yeah appveyor doesnt uses vcxproj. There is an option for static linking of the CRT which is not used.
To quickly check if your build is statically linked the dll weight around 150kB.

**Comment by dantarion** [comment]
https://github.com/dantarion/SF5DInput/releases/tag/0.0.8
I think I got it fixed now.

**Comment by michaelmarcuccio** [comment]
Looks great. Everything functioning so far!

**Comment by WydD** [comment]
Great it's just missing the tournament edition target and we're good to go
